share/settings/remote.go

```diff
@@ -85,7 +85,7 @@ func DecodeRemote(s string) (*Remote, error) {
 			return nil, errors.New("Missing ports")
 		}
 		if !isHost(p) {
-			return nil, errors.New("Invalid host")
+			return nil, errors.New("Invalid host" + p)
```

Comment:
this could return `fmt.Errorf("Invalid host (%s)", p)`

Reply:
My bad, forgot to remove this after doing some testing.